openproblems/tasks/denoising/methods/alra.py

```diff
@@ -29,7 +30,7 @@ def alra(adata, test=False):
     adata.obsm["train_norm"], libsize = scprep.normalize.library_size_normalize(
         adata.obsm["train_norm"], rescale=1, return_library_size=True
     )
-    adata.obsm["train_norm"] = adata.obsm["train_norm"].tocsr()
+    adata.obsm["train_norm"] = scipy.sparse.csr_matrix(adata.obsm["train_norm"])
```

Comment:
This doesn't seem like the right way to go. What was the error message? If it's not coming out of scprep as a spmatrix we should fix that.